bin/dot

```diff
@@ -65,7 +65,6 @@ brew update
 
 # Install homebrew bundle
 msg "🍺" "Brew install bundle from .Brewfile." 
-brew tap "homebrew/core" 2>&1
```

Comment:
what was breaking about this?

Reply:
It's builtin now, and they advise not using it or using a --force flag. I opted for letting it go as I don't see a good reason to keep it.